versioned_docs/version-latest/protocol/substrate-cli.md

```diff
@@ -0,0 +1,767 @@
+---
+title: Advanced CLI
+sidebar_position: 3
+description: How to run a Subspace Network Farmer with the Substrate CLI
+keywords:
+    - Farmer
+    - Farming
+    - CLI
+    - Binaries
+    - Docker
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Link from '@docusaurus/Link';
+import styles from '@site/src/pages/index.module.css';
+
+
+:::caution Advanced Usage
+[Subspace CLI](https://github.com/subspace/subspace-cli) is the recommended way to farm on the Subspace Network. Follow the guide below to get started or check out the project [README on GitHub](https://github.com/subspace/subspace-cli/blobl/main/README.md).
```

Comment:
The last link has a typo
`https://github.com/subspace/subspace-cli/blob/main/README.md`

Reply:
The same link is also broken for the pre-release version, please fix it there too 🙏

Reply:
Resolved in commit 97dd5cf & 9a76b88